lib/config.js

```diff
@@ -5,6 +5,10 @@ module.exports = {
 		host: 'https://api.paperspace.io',
 		logHost: 'https://logs.paperspace.io',
 	},
+	development: {
+		host: 'https://dev-api.paperspace.io',
+		logHost: 'https://logs-dev.paperspace.io',
```

Comment:
We try not to check these values in because this is a public repo.  We are trying not to expose those hosts to outside users.

Also we've had end-users run into problems because they often are using NODE_ENV=development or test.  I don't have a complete solution for that.

Can you drop this change for now and keep it as a local override until we have a better system?